cardInfoAPI-Level1.yaml

```
@@ -0,0 +1,1402 @@
+openapi: 3.1.0
+info:
+  version: 0.7.0
+  title: Common Card API (Public)
+  description: >
+    This specification defines a common card API for payment cards used in Switzerland.
+    The API is supposed to be used by third parties to retrieve payment card information for scheme-based debit, credit or prepaid cards (read only).
+  contact:
+    email: info@common-api.ch
+  license:
+    name: Apache 2.0
+    url: https://www.apache.org/licenses/LICENSE-2.0.html
+servers:
+  - url: https://card.common-api.ch
+externalDocs:
+  description: Find out more about SFTI API specifications
+  url: https://www.common-api.ch
+
+tags:
+  - name: Cards
+    description: Operations for retrieving information about payment cards.
+  - name: Card-tokens
+    description: Operations for retrieving information about tokens associated with payment cards.
+  - name: Transactions
+    description: Operations for retrieving information about transactions and authorizations done with payment cards or tokens.
+
+security:
+  - ApiKeyAuth: []
+  - OAuth2:
+      - read
+      - write
+
+paths:
+  /cards:
+    get:
+      summary: Provides a list of cards which match the provided query parameters.
+      description: >
+        This operation enables the retrieval of a list of payment cards. Payment cards can be physical or virtual cards.
+        In order to prevent API consumers from having to fulfill PCI DSS requirements, no full PAN or other data relevant for this regulation are included.
+        The API consumer is expected to have a non-card related identifier for the customer for whom data should be returned (e.g. card account number, bank account number).
+      tags:
+        - Cards
+      parameters:
+        - in: query
+          name: contract_reference
+          required: false
+          schema:
+            $ref: '#/components/schemas/ContractReference'
+          description: Retrieves all cards related to a contract. Value has to be a functional reference to the contract that can be known by the end customer.
+        - in: query
+          name: person_reference
```

What are the naming convention for query parameter, respectively why do we use different styles like snake_case in the cards API and lower camelCase in the accounts API?


```suggestion
          name: personReference
```

---

Thank you @dinaschoch for this good spot about consistency in our specification, and we already addressed this topic. 
The naming convention for query parameters is snake_case ([SFTI naming conventions](https://github.com/swissfintechinnovations/.github/wiki/Naming-Conventions), query-parameter). The naming of the account API will be aligned in the next major release, because this is a breaking change in the API. After a short migration phase, all specifications will have the same naming.